examples/course-minimal/qwiklabs.yaml

```diff
@@ -33,11 +33,14 @@ prerequisites:
     en: <p>A basic understanding of serverless architecture is recommended but not required.</p>
 
 tags: [sample, life-changing, gcp]
+gcp_product_tags: ['bigquery']
```

Comment:
Can we make this more generic (maybe `platform_product_tags`) so people can write `['bigquery']` or `['ec2', 's3']` or `['cuda', 'jupyter']`?

Reply:
Also, do we have a whitelist for these tags or can they be anything?

Reply:
Updated to `product_tags`. We have a whitelist - it is in the spreadsheet linked above in the attribute table.